react/components/Sidebar.js

```diff
@@ -216,8 +212,8 @@ class Sidebar extends Component {
                                 {
                                   this.state.page === 'home' && (
                                     <div>
+                                      <SimulatorMenuItem />
```

Comment:
Why did we decide to encapsulate it?

Reply:
Our `SimulatorMenuItem` differs from Cartman's original `MenuItem`. It does not navigate to a new Cartman's page, but works like an anchor to a new window, with a quite specific label (text + external link icon on hover).

Reply:
I could have changed the current MenuItem to support both scenarios, but it doesn't worth for this single case